configure.ac

```diff
@@ -681,6 +690,7 @@ AC_CONFIG_FILES([Makefile				\
 		 man/man1/pkcsep11_migrate.1		\
 		 man/man1/pkcsep11_session.1		\
 		 man/man1/pkcstok_migrate.1		\
+		  man/man1/pkcsstats.1			\
```

Comment:
Just my personal opinion, but we provide configure-switches for all tools (why?), but unconditionally build the man-pages for all tools.  That seems weird to me.

Reply:
Yes that's somewhat inconsistent. But at least we only include the built man page according to the configure-switch in man1.mk....